src/pykeen/evaluation/evaluator.py

```diff
@@ -559,10 +569,16 @@ def evaluate(
     if filtering_necessary or positive_masks_required:
         if additional_filtered_triples is None:
             logger.warning(
-                'filtered setting was enabled, but there were no `additional_filtered_triples`.'
-                ' This means you probably forgot to pass the training triples.',
+                'Filtered setting was enabled, but there were no `additional_filtered_triples`.'
+                ' This means you probably forgot to pass (at least) the training triples.'
+                ' Try:\n\n\tadditional_filtered_triples=[dataset.training.mapped_triples]\n\n'
+                'Or if you want to use the Bordes et al. (2013) approach to filtering,'
+                ' do:\n\n\tadditional_filtered_triples=[dataset.training.mapped_triples,'
+                ' dataset.validation.mapped_triples]\n',
```

Comment:
Use `dedent("""....""")` with "written out" line breaks instead to make this longer string more readable in source code.

Reply:
good point

Reply:
Done in 16b6a03f